src/applications/representative-appoint/tests/pages/representative/selectAccreditedRepresentative.unit.spec.jsx

```diff
@@ -99,7 +99,7 @@ describe('<SelectAccreditedRepresentative>', () => {
     });
 
     context('when the search input is valid', () => {
-      it('sets the results in state', async () => {
```

Comment:
I suspect this test is causing issues because the prop name didn't change in the test setup. It still has `view:representativeQuery`.

Reply:
Thanks, those snuck back in. Fixed with the latest.